src/test/java/teammates/ui/webapi/GetFeedbackResponsesActionTest.java

```diff
@@ -145,6 +145,30 @@ protected void testExecute_commentSubmission_shouldGetCommentsSuccessfully() thr
         verifyFeedbackCommentEquals(comment1FromStudent1, actualResponses.get(0).getGiverComment());
     }
 
+    @Test
+    protected void testExecute_getTextQuestionType_shouldGetCommentsSuccessfully() throws Exception {
+        DataBundle dataBundle = loadDataBundle("/FeedbackResponseCommentCRUDTest.json");
+        removeAndRestoreDataBundle(dataBundle);
+
+        InstructorAttributes instructor1OfCourse1 = dataBundle.instructors.get("instructor1OfCourse1");
+        FeedbackQuestionAttributes qn2InSession1 = dataBundle.feedbackQuestions.get("qn2InSession1");
+        FeedbackResponseAttributes response1ForQ2 = dataBundle.feedbackResponses.get("response1ForQ2");
+        FeedbackResponseCommentAttributes comment1FromInstructor1Question2 =
+                dataBundle.feedbackResponseComments.get("comment1FromInstructor1Question2");
```

Comment:
Given that the aim is for students to be able to comment on their responses, I think we should change this test to be a comment from a student instead?